src/pint/models/noise_model.py

```diff
@@ -307,9 +301,7 @@ def __init__(
                 name="ECORR",
                 units="us",
                 aliases=["TNECORR"],
-                description="An error term added that"
-                " correlated all TOAs in an"
-                " observing epoch.",
+                description="An error term that correlates all TOAs in an observing epoch.",
```

Comment:
Perhaps "An error term that is correlated among all TOAs in an observing epoch"?

Reply:
changed.